client/scripts/directives/field.js

```diff
@@ -212,13 +212,33 @@ require('../app').directive('field', /* @ngInject */function ($q, Raven, geoloca
 
         case 'select':
         case 'checkbox-group': {
+          // set viewModel as empty string for default value, otherwise typeahead filter is not working
+          field.viewModel = ''
+          $scope.$watch('field.model', function () {
+            if (!field.model) {
+              field.viewModel = ''
+              return
+            }
+
+            field.values.forEach(function (value) {
```

Comment:
use `[].every` or `for ... of ...` to break out of loop. Better if using a map for direct lookup